contracts/cw721-receiver/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "cw721-receiver"
+description = "A basic contract that can receive a cw721 token for testing purposes"
+authors = ["Christoph Otter <chris@confio.gmbh>"]
+version = { workspace = true }
+edition = { workspace = true }
+license = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+documentation = { workspace = true }
+rust-version = { workspace = true }
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[features]
+# for more explicit tests, cargo test --features=backtraces
+backtraces = ["cosmwasm-std/backtraces"]
+# use library feature to disable all instantiate/execute/query exports
+library = []
+
+[dependencies]
+cosmwasm-schema = { workspace = true }
+cosmwasm-std = { workspace = true }
+cw-storage-plus = { workspace = true }
+cw2 = { workspace = true }
+cw721 = { workspace = true }
+cw721-base = { workspace = true, features = ["library"] }
```

Comment:
Only used in tests, so can move to the dev-deps?

Reply:
I moved cw721-base. cw721 is needed because I reuse the `Cw721ReceiveMsg` in the contract's execute msg.